app/controllers/posts_controller.rb

```diff
@@ -1,12 +1,12 @@
 # Posts controller class extends application controller
 class PostsController < ApplicationController
   def index
-    @user = User.find(params[:user_id])
+    @user = User.find(params[:author_id])
     @posts = @user.posts.includes(:comments)
   end
 
   def show
-    @user = User.find(params[:user_id])
+    @user = User.find(params[:author_id])
     @post = Post.find(params[:id])
     @comments = @post.comments
```

Comment:
- [ ] I am not able to create a post because you didn't add the `create` method
<img width="1439" alt="Screen Shot 2023-03-10 at 09 56 39" src="https://user-images.githubusercontent.com/44745855/224256984-9592d84a-a92b-47f2-ab1e-b4513d56f477.png">